lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
@scottnath Pubcentral is using Drupal.org's eslint configs, which insists on containing `'use strict';` in functions. I've been experimenting with the Drupal eslint configs myself:

➜  pattern-library-utilities git:(scottnath-add-importer) eslint --config ../eslintrc-drupal lib/get-file-paths.js

/Users/Hyunho/Sites/PatternLibrary/pattern-library-utilities/lib/get-file-paths.js
   1:1   error    Use the function form of "use strict"  strict
   5:1   warning  Missing JSDoc @returns for function    valid-jsdoc
  10:24  error    Use the function form of "use strict"  strict
  19:2   error    Missing semicolon                      semi

Once you put `'use strict';` in a function, eslint stops complaining. We should keep enforcing strict mode, just within functions. BTW, eslinting this pr will output hundreds of lines of suggestions. If you want to do this, copy over the configs you referenced, and add `"node": true` to the `env` property.